### PR TITLE
Added PackedScene example under Resource types

### PR DESCRIPTION
**Pull request by Tybobobo**
Added PackedScene example under Resource types, as per request

### CODE REVIEW

File: reference/gdscript.rst [hunk 1]
```diff
@@ -799,9 +799,12 @@ special export syntax is provided.
     export(int) var number
 
     # Export can also take a resource type to use as a hint
-
+
```
Comment (akien-mga) [inline reference/gdscript.rst]: You've added trailing spaces here :)

File: reference/gdscript.rst [hunk 2]
```diff
@@ -799,9 +799,12 @@ special export syntax is provided.
...
 
     # Export can also take a resource type to use as a hint
-
+    
+    # Select Texture image
     export(Texture) var character_face
-
+    # Select a scene file from your project
+    export (PackedScene) var scene_file
+
```
Comment (akien-mga) [inline reference/gdscript.rst]: And there.

Comment (Tybobobo) [inline reference/gdscript.rst]: Yes, to make it consistent with the rest of the code structure and commenting.
http://i.imgur.com/oSCvaNV.png

Comment (akien-mga) [inline reference/gdscript.rst]: I don't see how that brings any consistency?
image

**Comment by akien-mga** [comment]
Superseded by 7837e4b1.

**Pull request closed without merging**